theano/tensor/nnet/tests/test_nnet.py

```diff
@@ -1295,7 +1309,7 @@ def test_argmax_pushdown():
         fgraph = gof.FunctionGraph(
                 [x],
                 [out])
-        assert hasattr(fgraph.outputs[0].tag, 'trace')
+        assert check_stack_trace(fgraph, ops_to_check='all')
```

Comment:
For that test, I think it is actually the previous `fgraph` that should be checked. We can check that one as well with 'all' since the optimization is not supposed to apply...
For the first `fgraph` above, you can check for `tensor.basic._max_and_argmax`.